Xamarin.Forms.Core/Interactivity/VisualStateTrigger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms
+{
+	public class VisualStateTrigger : TriggerAction<VisualElement>
+	{
+		public string GoToState { get; set; }
```

Comment:
I think this property should be named `State`.

Reply:
done